edx_sga/sga.py

```diff
@@ -705,6 +804,47 @@ def file_storage_path(self, sha1, filename):
         )
         return path
 
+    def get_user(self):
+        """returns user object"""
+        try:
+            user = self.runtime.get_real_user(self.xmodule_runtime.anonymous_student_id)
+        except Exception:  # pylint: disable=broad-except
+            user = None
+
+        return user
+
+    def is_zip_file_available(self, user):
+        """
+        returns True if zip file available.
+
+        Args:
+            user (django.contrib.auth.models.User): user instance
+        """
+        if user:
+            zip_file_path = get_zip_file_path(
+                user.username,
+                self.block_course_id,
+                unicode(self.block_id),
+                self.location
+            )
+            return True if os.path.exists(zip_file_path) else False
+        return False
+
+    def start_task(self, user):
+        """
+        Starts submission compression task
+        """
+        if user:
+            zip_student_submissions.delay(
+                self.block_course_id,
+                self.block_id,
+                self.location,
+                user
+            )
+        return Response(json_body={
+            "downloadable": False
+        })
```

Comment:
As I mentioned in [this comment](https://github.com/mitodl/edx-sga/pull/187#discussion_r155649601), I still think this function should be removed